packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -18,8 +17,34 @@ export class WalletController implements ReactiveController {
 
   walletContext: ContextConsumer<typeof walletContext, ReactiveElement>;
 
+  /**
+   * Provides list of wallets specified by 3rd party
+   * along with default injected connector
+   * @param {{ dappUrl?: string }} options
+   * @returns {WalletInit[]}
+   */
+  getWallets(options?: {
+    walletConnectOptions?: WalletConnectOptions;
+    walletSelectorWalletConfigurations?: WalletInit[];
+    appMetaData?: AppMetadata;
+  }): WalletInit[] {
+    // always have injected ones
+    const injected = injectedModule();
+    let wallets = [injected];
+
+    if (
+      options?.walletSelectorWalletConfigurations &&
+      options?.walletSelectorWalletConfigurations.length > 0
+    ) {
+      wallets = wallets.concat(options.walletSelectorWalletConfigurations);
+    }
+
+    return wallets;
```

Comment:
I'd maybe rather use `const` if possible + a bit shorter `if` condition

```suggestion
    const wallets = [injected];

    if (options?.walletSelectorWalletConfigurations?.length) {
      wallets.push(...options.walletSelectorWalletConfigurations);
    }

    return wallets;
```